install.sh

```diff
@@ -136,18 +165,7 @@ ohai "Downloading and installing Bugsnag CLI..."
 (
   cd "${BUGSNAG_CLI_PREFIX}" >/dev/null || return
 
-  DOWNLOAD_URL=$(curl -sS ${BUGSNAG_CLI_GIT_REMOTE} |
-    grep "${UNAME_MACHINE}-${OS_NAME}-bugsnag-cli*" |
-    grep "browser_download_url" |
-    cut -d : -f 2,3 |
-    tr -d \" |
-    xargs)
-
-  if [[ ! -n ${DOWNLOAD_URL} ]]; then
-    abort "Failed to get download URL from ${BUGSNAG_CLI_GIT_REMOTE}"
-  fi
-
-  execute "curl" "-#" "-L" "${DOWNLOAD_URL}" "-o" "${BUGSNAG_CLI_PREFIX}/bin/bugsnag-cli"
+  execute "curl" "-#" "-L" "https://github.com/bugsnag/bugsnag-cli/releases/download/v${VERSION}/${UNAME_MACHINE}-${OS_NAME}-bugsnag-cli" "-o" "${BUGSNAG_CLI_PREFIX}/bin/bugsnag-cli"
```

Comment:
Is there a way to check whether the download succeeded? If you enter a non-existent version, it's not clear from the output that nothing happened.